src/main/scala/org/reactivestreams/Publisher.scala

```diff
@@ -0,0 +1,29 @@
+package org.reactivestreams
```

Comment:
As long as there is no clash I don't see any real problem with it. @reactive-streams/contributors, thoughts?